terraform/modules/enclave/README.md

```diff
@@ -0,0 +1,33 @@
+# Prometheus verify enclave
+
+The enclave environment is reference to the area of verify that we use to perform  our deployment. This environment has a number of different security requirements that difficult and a very custom environment to deploy too.
```

Comment:
I'm not sure if the second sentence here is adding much value so maybe could be either deleted or expanded on to be specific (although feel free to disagree). I think there is a missing word too before `difficult`.